lib/index.js

```diff
@@ -25,15 +27,28 @@ function initServer() {
     return server;
 }
 
-function initLogger() {
+function initLogger(abbreviation) {
     const bunyan = require('bunyan');
+    // initialize maximum body logging length
+    bodyAbbreviationLength = abbreviation;
     return bunyan.createLogger({
         name: packageJson.name
     });
 }
 
 function logJson(logger, body) {
-    logger.info(JSON.stringify(body, null, 4));
+    // Make a copy so the original body doesn't get mutated
+    const content = Object.assign({}, body);
+    // If the body is too long of a string, abbreviate it
+    if (bodyAbbreviationLength > 10 && content.body && content.body.length && content.body.length > bodyAbbreviationLength) {
+        const halfAbbrev = bodyAbbreviationLength / 2;
```

Comment:
Speaking of `--abbrev 5`, that would make `halfAbbrev` come out as 2.5.  I just tried it and it turns out that works fine--it seems to drop the decimal part internally, at least in the Node repl--but it still seems like we should call `Math.floor()` on it rather than counting on that.